### ISSUE TITLE
askForContact() function doesn't work

### ISSUE THREAD

**Issue by no4ch** [mpociot/botman#1171]
- BotMan Version: 2.4
- Driver Telegram: 1.5
- PHP Version: 7.2
- Messaging Service(s):
- Cache Driver:

### Description:

I created a new class _LoginConversation_ , but method askForContact() doesn't work
and every time it asks again and again for the phone

 public function run()
    {
        $this->askContact();
    }

    public function askContact()
    {
        $this->askForContact('Please send us your phone number.', function (Contact $contact) {
            //$this->askEmail();
            $this->say('Thank you. Your contact has been linked.');
        },
            null,
            $this->keyboard()
        );

    }

    public function keyboard()
    {
        return Keyboard::create()
            ->addRow(KeyboardButton::create('Отправить свой номер')->requestContact())
            ->type(Keyboard::TYPE_KEYBOARD)
            ->oneTimeKeyboard()
            ->resizeKeyboard()
            ->toArray();
    }
![image](https://user-images.githubusercontent.com/49817056/87782345-06be9380-c83b-11ea-86c9-bf56cb49c0e0.png)

**Comment by PrinceofMath** [comment]
check logs

**Comment by no4ch** [comment]
> check logs

there were no errors in the logs, it's just that the function is endless and constantly asks for a phone number, while when the server receives it, it asks again

**Comment by PrinceofMath** [comment]
In TelegramContactDriver change loadMessages():
```php
public function loadMessages()
    {
        $message = new IncomingMessage(Contact::PATTERN, $this->event->get('from')['id'], $this->event->get('chat')['id'], $this->event);
        $message->setContact(new Contact(
            $this->event->get('contact')['phone_number'],
            $this->event->get('contact')['first_name'] ?? '',
            $this->event->get('contact')['last_name'] ?? '',
            $this->event->get('contact')['user_id'] ?? '',
            $this->event->get('contact')['vcard'] ?? ''
        ));

        $this->messages = [$message];
    }
```

**Comment by no4ch** [comment]
> In TelegramContactDriver change loadMessages():
> public function loadMessages()
> {
> $message = new IncomingMessage(Contact::PATTERN, $this->event->get('from')['id'], $this->event->get('chat')['id'], $this->event);
> $message->setContact(new Contact(
> $this->event->get('contact')['phone_number'],
> $this->event->get('contact')['first_name'] ?? '',
> $this->event->get('contact')['last_name'] ?? '',
> $this->event->get('contact')['user_id'] ?? '',
> $this->event->get('contact')['vcard'] ?? ''
> ));
> 
> ```
>     $this->messages = [$message];
> }
> ```

in vendor??

**Comment by PrinceofMath** [comment]
Yeah I did it and it works.

**Comment by no4ch** [comment]
> Yeah I did it and it works.

I think it's worth inheriting and redefining the method, since the "vendor" is the same for everyone

**Comment by PrinceofMath** [comment]
> > Yeah I did it and it works.
> 
> I think it's worth inheriting and redefining the method, since the "vendor" is the same for everyone


Yeah. Is it solved your problem?

**Comment by vdomah** [comment]
Fixed in btoman/driver-telegram v1.6.2
https://github.com/botman/driver-telegram/commit/102cd6de52cefbe3f119a6b25ee924cc72bcfb77

**Comment by paulvales** [comment]
Hello! no callback is called in askForContact, inserted logging into it and nothing happens. cache cleared.

- BotMan Version: 2.6.1
- Driver Telegram: 1.6.2
- PHP Version: 7.2
- Cache Driver: Doctrine\Common\Cache\PhpFileCache

![image](https://user-images.githubusercontent.com/13873237/113762281-3fd8c400-973a-11eb-9f33-78296c2bb2eb.png)
